<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCreateClass</class>
 <widget class="QMainWindow" name="QtCreateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCreate</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/QtCreate/Logo.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ScheduleGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>调度策略</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="EDFpushButton">
        <property name="text">
         <string>相对短截止时间优先调度(EDF)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RMSpushButton">
        <property name="text">
         <string>单调速率调度(RMS)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DMSpushButton">
        <property name="text">
         <string>短截止时间调度(DMS)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LSTpushButton">
        <property name="text">
         <string>最小松弛时间调度(LST)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MFPSpushButton">
        <property name="text">
         <string>最小空闲时间共享分区调度(MFPS)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ConsoleGroupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>531</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>实时调度信息</string>
    </property>
    <widget class="QTextEdit" name="ConsoleTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>511</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GantGroupBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>591</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>运行状态图</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Task</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Timeslice</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DataGroupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>调度数据</string>
    </property>
    <widget class="QTextEdit" name="DataTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ResultGroupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>300</y>
      <width>361</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>运行结果</string>
    </property>
    <widget class="QTextEdit" name="ResultTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="FileOpenButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>580</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FileOpen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FileSaveButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>FileSave</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>121</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/QtCreate/logo8.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtcreate.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>FileOpenButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>LoadAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>615</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FileSaveButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>SaveAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>645</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LSTpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>LSTBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RMSpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>RMSBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MFPSpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>MFPSBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EDFpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>EDFBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DMSpushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtCreateClass</receiver>
   <slot>DMSBtn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>EDFBtn_Click()</slot>
  <slot>RMSBtn_Click()</slot>
  <slot>DMSBtn_Click()</slot>
  <slot>LoadAction()</slot>
  <slot>SaveAction()</slot>
  <slot>MFPSBtn_Click()</slot>
  <slot>LSTBtn_Click()</slot>
 </slots>
</ui>
